mbodied/types/sense/scene.py

```diff
@@ -0,0 +1,42 @@
+from typing import List
+
+from pydantic import Field
+
+from mbodied.types.geometry import Pose
+from mbodied.types.ndarray import NumpyArray
+from mbodied.types.sample import Sample
+from mbodied.types.sense import Image
+
+
+class SceneObject(Sample):
+    """Model for Scene Object Poses.
+
+    Attributes:
+        name (str): The name of the object.
+        pose (Pose): The pose of the object.
+        bbox_3d (NumpyArray | None): The 3D bounding box of the object.
+        pixel_coords (NumpyArray | None): The pixel coordinates of the object.
+        bbox_pixels (NumpyArray | None): The pixel coordinates of the bounding box.
+    """
+
```

Comment:
```suggestion
class BBox2D(NamedTuple):
  x1: float
  y1: float
  x2: float
  y2: float
```
PixelCoords(NamedTuple):
  u: int
  v: int

BBox3D... etc